### PR TITLE
Resolving the issue of remaining highlights

### PR DESCRIPTION
**Pull request by FunamaYukina**
## Summary

I've resolved the problem where irrelevant TableNodes, Edges, and Cardinalities were being unexpectedly highlighted.
It often happens when moving the mouse pointer from node to node in an ERD with many tables.↖️

### Before

ss 2535

### After

ss 2536

## Testing

Please confirm if this issue still persists👀.
Preiew: https://liam-erd-sample-k8i0nsnmb-route-06-core.vercel.app/

### CODE REVIEW

**Comment by FunamaYukina** [comment]
I'm going to fix the conflict🙏.

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx [hunk 1]
```diff
@@ -164,44 +177,82 @@ export const ERDContentInner: FC<Props> = ({
...
+        (e) => e.source === activeNodeId || e.target === activeNodeId,
       )
 
+      const updatedEdges = edges.map((e) => {
+        if (relatedEdges.includes(e)) {
+          return highlightEdge(e)
+        }
+        if (relatedToActiveNodeEdges.includes(e)) {
+          return highlightEdge(e)
+        }
```
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx:190]: Is this the solution to the PR title cause?

**Review by MH4GF — APPROVED** [review]
The content looked good, please just resolve the conflicts!

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx (hunk 1 above)
Comment (FunamaYukina) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx:163]: I suspect that ``return unhighlightEdge(e)`` is referring to the code in the next line.
It seems that the ``unhighlightEdge`` function was not working correctly when the hovered TableNode changed.

**Comment by FunamaYukina** [comment]
I've checked the preview and it looks good👀, so I'm going to merge it🚀.
 Please let me know if you have any concerns🙏.

**Pull request merged**